app/models/concerns/orchestration/compute.rb

```diff
@@ -212,8 +212,9 @@ def delComputeUpdate
 
   def compute_update_required?
     return false unless compute_resource.supports_update? && !compute_attributes.nil?
-    old.compute_attributes = compute_resource.find_vm_by_uuid(uuid).attributes
-    compute_resource.update_required?(old.compute_attributes, compute_attributes.symbolize_keys)
+    attrs = compute_resource.find_vm_by_uuid(uuid).attributes
+    old.compute_attributes = attrs if old
```

Comment:
@houndci-bot: This is only possible with Ruby > Ruby 2.3.0. If I recall correctly the Foreman project supports older rubies.